main_app/models.py

```diff
@@ -46,3 +46,64 @@ class UserModel(PermissionsMixin, AbstractBaseUser):
 
     def __str__(self):
         return self.email
+
+
+class MustModel(models.Model):
+    game_id = models.IntegerField()
```

Comment:
Should be ForeignKey to `GameModel` (probably views should be updated according to this)